docs/hello-neighbour.yaml

```diff
@@ -184,212 +224,130 @@ paths:
-        name: "churchUuid"
+      - name: "churchUuid"
+        in: "path"
+        required: true
         type: "string"
         format: "uuid"
-        required: true
       responses:
         200:
           description: "Question list"
           schema:
             type: "array"
             items:
               $ref: "#/definitions/Question"
+        401:
+          $ref: "#/responses/Unauthenticated"
+        403:
+          $ref: "#/responses/Unauthorized"
         404:
           description: "Church not found"
           schema:
-            $ref:  "#/definitions/ErrorResponse"
+            $ref:  "#/definitions/Error"
 
-  /church/question/{questionUuid}: # church is determined by requesting user's affiliation
+  /church/{churchUuid}/matchgroup: # church and user are determined based on user session
     get:
       tags:
       - "leader"
-      summary: "Retrieve question"
-      description: "Retrieve a question for church for given UUIDs."
-      operationId: "getQuestion"
-      parameters:
-      - in: "path"
-        name: "churchUuid"
-        type: "string"
-        format: "uuid"
-        required: true
-      - in: "path"
-        name: "questionUuid"
-        type: "string"
-        format: "uuid"
-        required: true
-      responses:
-        200:
-          description: "Church question"
-          schema:
-            $ref: "#/definitions/Question"
-        404:
-          description: "Question not found"
-          schema:
-            $ref:  "#/definitions/ErrorResponse"
-    put:
-      tags:
-      - "leader"
-      summary: "Update question"
-      description: "Update a question that's already in the system."
-      operationId: "modifyQuestion"
-      consumes:
-      - "application/json"
-      produces:
-      - "application/json"
-      parameters:
-      - in: "path"
-        name: "churchUuid"
-        type: "string"
-        format: "uuid"
-        required: true
-      - in: "path"
-        name: "questionUuid"
-        type: "string"
-        format: "uuid"
-        required: true
-      - in: "body"
-        name: "body"
-        description: "Question for the church"
-        required: true
-        schema:
-          $ref: "#/definitions/QuestionIn"
-      responses:
-        200:
-          description: "Question updated"
-          schema:
-            $ref: "#/definitions/Question"
-        404:
-          description: "Question not found"
-          schema:
-            $ref: "#/definitions/ErrorResponse"
-        400:
-          description: "Invalid input"
-          schema:
-            $ref: "#/definitions/ErrorResponse"
-    delete:
-      tags:
-      - "leader"
-      summary: "Delete question"
-      description: "Delete a question from the system."
-      operationId: "deleteQuestion"
+      summary: "Retrieve list of all your church match groups"
+      description: "Retrieve all match groups of a specific church (leader) or the ones you are currently part of (member)."
```

Comment:
Should we separate this as well? (see https://github.com/jsfan/hello-neighbour-api/pull/5/files#r420196809)

Reply:
I suppose we could restrict access to this endpoint and include the currently matched groups for a member in `/profile`.

Reply:
I restricted access to this endpoint and created a separate one for members since it doesn't seem to go with what `/profile` returns.